cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -950,12 +983,14 @@ search = `cs_wineventlog_security` (EventCode=4740) \
 | fields ComputerName Account_Domain Account_Name Caller_Computer_Name \
 | rename ComputerName as "Domain_Controller", Account_Domain as User_Domain ,Account_Name as "User_Name", Caller_Computer_Name as "Source_Computer_Name" \
 | fillnull value="-" Source_Computer_Name \
-| table _time,User_Name,User_Domain,Source_Computer_Name,Domain_Controller \
+| eval cyences_severity = "medium" \
```

Comment:
If we cannot determine whether user is high priority user or not in automated fashion let's create Jira ticket for the future.